ConsolidateArtifacts.bat

```diff
@@ -0,0 +1,27 @@
+setlocal EnableDelayedExpansion
+
+:: Libraries from local folder
+@SETLOCAL
+@ECHO off
+call "%VS140COMNTOOLS%..\..\VC\bin\vcvars32.bat"

```

Comment:
I am now using 64-bit visual studio compiler but the blosc project doesn't provide any configuration properties for building in 64-bit, it's only provided for Win32.
Also added clean up of temporary files and folders used for building